amplipi/models.py

```diff
@@ -135,6 +135,13 @@ class SourceInfo(BaseModel):
   station: Optional[str]  # name of radio station
   img_url: Optional[str]
   supported_cmds: List[str] = []
+  rating: Optional[int]
+  # "rating" is a pandora-specific enum
+  # decode where text is common wording and ("text") is the pandora/pianobar terminology
+  # 0 default
+  # 1 liked ("loved")
+  # 2 disliked ("banned")
+  # 3 shelved ("tired")
```

Comment:
can you use Python 3's `enum` here?

Reply:
I don't really understand `enum` but adding it as I just did seems to work after testing so I guess that's fine to do
I left the comment block in as it still belongs to translate terminology as well as the use for the "rating" variable